workflow-ui/core-android/src/main/java/com/squareup/workflow1/ui/RealScreenViewHolder.kt

```diff
@@ -0,0 +1,20 @@
+package com.squareup.workflow1.ui
+
+import android.view.View
+
+@WorkflowUiExperimentalApi
+internal class RealScreenViewHolder<ScreenT : Screen>(
+  initialEnvironment: ViewEnvironment,
+  override val view: View,
+  viewRunner: ScreenViewRunner<ScreenT>
+) : ScreenViewHolder<ScreenT> {
+
+  private var _environment: ViewEnvironment = initialEnvironment
+  override val environment: ViewEnvironment get() = _environment
```

Comment:
nit: I think you could use `private set`:
```
override var environment: ViewEnvironment = initialEnvironment
  private set
```

Reply:
I can't, the public API is `val`. But if I drop the interface, then hell yeah.